KIPs/kip-fungible_token.md

```diff
@@ -0,0 +1,250 @@
+---
+kip: 7
+title: Fungible Token Standard
+author: Junghyun Colin Kim <colin.kim@groundx.xyz>, Kyungup Kim <ethan.kim@groundx.xyz>
+discussions-to: <URL>
```

Comment:
Please create an issue in this repo and put the URL here.
This issue is for discussion of this KIP.